app/test_main.py

```diff
@@ -1,3 +1,70 @@
 from app.main import get_human_age
+import pytest
+from typing import Any
 
-# write your code here
+
+class TestValues:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,expected_value",
+        [
+            pytest.param(
+                15,
+                15,
+                [1, 1],
+                id="Correct values for 15, 15"),
+            pytest.param(
+                24,
+                24,
+                [2, 2],
+                id="Correct values for, 24, 24"),
+            pytest.param(
+                28,
+                29,
+                [3, 3],
+                id="Correct values for 28, 29"),
+            pytest.param(
+                0,
+                0,
+                [0, 0],
+                id="Return 0 when animal year is 0"),
+            pytest.param(
+                0,
+                0,
+                [0, 0],
+                id="Return 0 when animal age is 0"),
```

Comment:
There are two identical test cases for zero values (lines 25-29 and 30-34). According to the checklist, you should avoid testing the same case multiple times. Consider removing one of these test cases to adhere to the DRY principle.